network-api/networkapi/wagtailpages/models.py

```diff
@@ -443,6 +443,7 @@ class PrimaryPage(FoundationMetadataPageMixin, Page):
         on_delete=models.SET_NULL,
         related_name='primary_banner',
         verbose_name='Hero Image',
+        help_text='choose an image that\'s bigger than 4032px x 1152px with aspect ratio 3.5:1',
```

Comment:
Most likely me just being picky, can we capitalize the 'c' in 'choose'?

Reply:
Also, would this help text mainly be for designers? Only because I don't think most would be that familiar with aspect ratio.

Reply:
yes help text is mainly be for designers